### PR TITLE
Fixed markdown issue in PROCESS.md

### PR DESCRIPTION
**Pull request by jvanbaarsen**
/cc @123Haynes Can you take a look?

### CODE REVIEW

File: PROCESS.md [hunk 1]
```diff
@@ -119,6 +119,6 @@ rebase with master to see if that solves the issue.
 ### Closing down the issue tracker on GitHub
 
 We are currently in the process of closing down the issue tracker on GitHub, to
-prevent duplication with the [GitLab.com issue tracker][https://gitlab.com/gitlab-org/gitlab-ce/issues].
+prevent duplication with the [GitLab.com issue tracker](https://gitlab.com/gitlab-org/gitlab-ce/issues).
```
Comment (Razer6) [inline PROCESS.md]: @jvanbaarsen Please escape the braces to make the text to work with copy&paste. Right now those links are rendered has links.

Comment (jvanbaarsen) [inline PROCESS.md]: Good point!

**Comment by jvanbaarsen** [comment]
@Razer6 Fixed

**Comment by Razer6** [comment]
I would only use the last `GitLab.com issue tracker` as an issue.

**Comment by Razer6** [comment]
It's both the same link and people would already click on the first one before continuing reading.

**Comment by jvanbaarsen** [comment]
@Razer6 Hm, you I think you're right! Will update later today

**Comment by Razer6** [comment]
@jvanbaarsen Ping me and I merge then

**Comment by jvanbaarsen** [comment]
@Razer6 Updated (sorry for delay)

**Pull request merged**